db/seeds.rb

```diff
@@ -82,7 +82,8 @@
 Vacancy.listed.each do |vacancy|
   statuses = JobApplication.statuses.keys
   Jobseeker.where.not(email: "jobseeker@example.com").each do |jobseeker|
-    application_status = statuses.delete(statuses.sample)
```

Comment:
This reduces the list one by one. It started returning `nil` after going through all the values once the amount of seeded Jobseekers was greater than the list of possible statuses.